docs/source/teams/roles_and_permissions.rst

```diff
@@ -166,22 +171,23 @@ following sources:
 
 .. _teams-default-access:
 
-Default access
---------------
+Implicit access
+---------------
 
-All datasets have a default access level, which defines a minimum permission
-level that all Members have on the dataset.
+All datasets have a implicit access level, previously known as the default 
+access level, which defines a minimum permission level for members with 
+implicit data access roles. 
 
-A dataset's default access level can be set to **No access**, **Can view**,
-**Can edit**, or **Can manage** as shown below:
+A dataset's implicit access level can be set to **No access**, **Can view**,
+**Can tag**, **Can edit**, or **Can manage** as shown below:
 
 .. image:: /images/teams/dataset_default_access.png
-   :alt: default-access
+   :alt: implicit-access
    :align: center
 
 .. note::
 
-   Default access level only applies to Members. Guests and Collaborators must
+   Implicit access level only applies to Members. Guests and Collaborators must
```

Comment:
Can we please change the `People and groups with access` heading back to `Specific access` below?

Here I'm proposing to revert a change that was made when user groups were introduce so that the table of contents matches the jargon in the App.